core/src/components/picker-column/test/standalone/picker-column.e2e.ts

```diff
@@ -0,0 +1,19 @@
+import { test } from '@utils/test/playwright';
+
+import { testPickerColumn } from '../test.utils';
+
+test.describe('picker-column', () => {
```

Comment:
Can we merge the screenshot tests in first so we can better check for visual regressions? I am noticing that removing `contain: content` causes the fonts to render bigger than before in WebKit:


| with `contain: content` | without `contain: content` |
| ------- | ------- |
| ![IMG_0775](https://user-images.githubusercontent.com/2721089/168816531-2258e029-c187-49d3-8f92-92c7be4775b4.PNG) | ![IMG_0776](https://user-images.githubusercontent.com/2721089/168816566-0bd17395-5d1e-441f-9ea7-55b4eb7b5508.PNG) |

However, I am not sure if this is another bug or how the browser should be behaving.